e2e/playwright/regression/shared/config.ts

```diff
@@ -0,0 +1,76 @@
+import { Page, Expect } from '@playwright/test';
+
+import * as uuid from "uuid";
+
+export const validateInitialConfig = async (page: Page, expect: Expect) => {
+  const sidebar = page.getByTestId('config-sidebar-wrapper');
+  await expect(sidebar).toBeVisible({ timeout: 15000 });
+  await expect(sidebar).toContainText('Nginx Config');
+  await expect(sidebar).toContainText('Nginx port');
```

Comment:
do all the regression tests use the same app?

Reply:
yeah